client/blocks/reader-subscription-list-item/style.scss

```diff
@@ -0,0 +1,64 @@
+
+.reader-subscription-list-item__by-text {
+	color: $gray;
+}
+
+.reader-subscription-list-item .gravatar {
+	float: left;
+	margin: 2px 6px 0 0;
+	height: 32px;
+	min-height: 32px;
+	width: 32px;
+	min-width: 32px;
+}
+
+.reader-subscription-list-item {
+	font-size: 14px;
+	padding-bottom: 20px;
+	display: flex;
+}
+// copy and pasted from follow button for reader-post-card
+// TODO: combing to reusable refreshed-reader component?
```

Comment:
oh yes, ill comb that comment off 🥇.  I think I was referring to how the default styles of follow-button are now prefresh and we could clean it up a bit by removing the prefresh css and having default be our new fresher follow buttons.

That can be a later janitorial PR though